File: Dreya/question/Question.java
package dreya.question;

/**
 *
 * @author smv
 */
public class Question {
    
    public boolean hasAnswer;
    public boolean hasRequestion = false;
    
        
    public String randomAnswer(String[] answers)
    {
        return answers[0];
    }
    
    public boolean hasAnswer()
    {
        return this.hasAnswer;
    }

    public String answer(String sender, String message, String[] normalized) {
        return ".";
    }


}

File: Dreya/Questions/Question.py
# -*- coding: utf-8 -*-

import random

class Question:
    normalized = []
    original = ""
    sender = ""

    def __init__(self, sender, normalized, original):
        self.normalized = normalized
        self.original = original
        self.sender = sender

    def randomAnswer(self, answers):
        return random.choice(answers)